### PR TITLE
docs: Update notifications (main) docs

### PR DESCRIPTION
**Pull request by codefoster**
#### Description of Change
Removed `nodeIntegration`
Linted code using `standard`.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes]

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by codefoster** [comment]
Just noticed that I had the wrong title on this PR (copy/paste error). Sorry for the confusion.

File: docs/tutorial/notifications.md [hunk 1]
```diff
@@ -55,18 +55,17 @@ Starting with a working application from the
...
-  }
-  new Notification(notification).show()
+  new Notification({ title: NOTIFICATION_TITLE, body: NOTIFICATION_BODY }).show()
 }
 
 app.whenReady().then(createWindow).then(showNotification)
 ```
 
-After launching the Electron application, you should see the notification:
+After launching the Electron application, you should see the system notification:
```
Comment (tonyferrell) [inline docs/tutorial/notifications.md:68]: idea: put this in index.html?

**Review by tonyferrell — APPROVED** [review]

File: docs/tutorial/notifications.md (hunk 1 above)
Comment (codefoster) [reply docs/tutorial/notifications.md:68]: Great point. Will do.

**Comment by codefoster** [comment]
@tonyferrell, gave the feedback on the _renderer_ version of the notifications feature that it's awkward to notify the user of success via the console since the presence of two consoles (the one in fiddle and the one in the running app dev tools) can be confusing. I agree. For this feature where the notification is generated in the `main.js` file, in order to update the UI, we have to jump through the IPC hoops. **Do you think we should, or should add that complexity to an example like this?** I'll implement it to see how much it adds.

File: docs/tutorial/notifications.md (hunk 1 above)
Comment (codefoster) [reply docs/tutorial/notifications.md:68]: Done in commit 8719

**Comment by tonyferrell** [comment]
> @tonyferrell, gave the feedback on the _renderer_ version of the notifications feature that it's awkward to notify the user of success via the console since the presence of two consoles (the one in fiddle and the one in the running app dev tools) can be confusing. I agree. For this feature where the notification is generated in the `main.js` file, in order to update the UI, we have to jump through the IPC hoops. **Do you think we should, or should add that complexity to an example like this?** I'll implement it to see how much it adds.

I think I agree that adding IPC to this probably overcomplicates the example.

**Comment by Ethan-Arrowood** [comment]
cc @erickzhao

**Review by erickzhao — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**